### PR TITLE
feat(new-trace): Fixes blank replay viewer in bottom docked drawer.

### PR DESCRIPTION
**Pull request by Abdkhan14**
This is for the replay viewer under the transaction tab in the trace drawer.

### CODE REVIEW

File: static/app/components/events/eventReplay/index.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ type Props = {
   group?: Group;
 };
 
-const CLIP_OFFSETS = {
+export const CLIP_OFFSETS = {
```
Comment (JonasBa) [inline static/app/components/events/eventReplay/index.tsx:34]: If we are exporting this, lets rename it to REPLAY_CLIP_OFFSET

File: static/app/views/performance/newTraceDetails/traceDrawer/details/transaction.tsx [hunk 2]
```diff
@@ -160,6 +164,41 @@ function BreadCrumbsSection({
...
+  organization: Organization;
+}) {
+  const replayId = getReplayIdFromEvent(event);
+  const startTimestampMS =
+    'startTimestamp' in event ? event.startTimestamp * 1000 : undefined;
+  const timeOfEvent = event.dateCreated ?? startTimestampMS ?? event.dateReceived;
+  const eventTimestampMs = timeOfEvent ? Math.floor(new Date(timeOfEvent).getTime()) : 0;
+
+  return (
+    <div style={{display: 'flex', flexDirection: 'column'}}>
```
Comment (JonasBa) [inline static/app/views/performance/newTraceDetails/traceDrawer/details/transaction.tsx:181]: Styled component please :)

File: static/app/views/performance/newTraceDetails/traceDrawer/details/transaction.tsx [hunk 3]
```diff
@@ -409,13 +448,14 @@ export function TransactionNodeDetails({
...
       {project ? <EventEvidence event={event} project={project} /> : null}
+      <ReplaySection event={event} organization={organization} />
       {event.projectSlug ? (
         <Entries
           definedEvent={event}
           projectSlug={event.projectSlug}
           group={undefined}
           organization={organization}
-          isShare={false}
+          isShare
```
Comment (JonasBa) [inline static/app/views/performance/newTraceDetails/traceDrawer/details/transaction.tsx:458]: Why are we changing this?

Comment (Abdkhan14) [reply static/app/views/performance/newTraceDetails/traceDrawer/details/transaction.tsx:458]: This prop set to true prevents the replay section from the Entries component from rendering. We have our own replay section now.

**Review by JonasBa — APPROVED** [review]

**Pull request merged**